src/components/page-list/page-list.jsx

```diff
@@ -44,20 +44,22 @@ export default class PageList extends React.Component {
   }
 
   renderPages () {
-    return (
-      <div className="row">
-        <div className="col-md-12">
-          <table className="table" styleName="page-list" >
-            <thead>
-              {this.renderHeader()}
-            </thead>
-            <tbody>
-              {this.props.pages.map(page => this.renderRow(page))}
-            </tbody>
-          </table>
+    if (this.props.isSearching) {
```

Comment:
Can you write up a quick test in `src/components/__tests__/page-list.test.jsx` to make sure the `Loading` component renders when `this.props.isSearching` is passed in?

Reply:
Hey @johnsonphan95, I think this might not be the best place for this check. The `PageList` component is designed to avoid having *too* much logic: if it’s given pages to render, it renders them, and if not, it shows the loading indicator (see lines 22-24).

It might make more sense to simply not give `PageList` any pages to show when search results are loading, rather than giving it extra props and logic to make it aware of the whole concept of searching. i.e. we should probably just be setting the `WebMonitoringUi` component’s `state.pages` to `null` while a search is loading new data. Does that make sense?

Reply:
Understood, I will go ahead and see what I can do!